tests/functional/pv/space_reclaim/test_staggered_cronjob_scheduling.py

```
@@ -302,6 +322,9 @@ def _collect_job_timestamps(
                 for cj_name, ts in timestamps.items():
                     logger.info("Job timestamp for CronJob '%s': %s", cj_name, ts)
                 return timestamps
+        raise TimeoutExpiredError(
+            f"Jobs not found for all CronJobs within {JOB_CREATION_TIMEOUT}s"
+        )
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)